dev-tools/mage/crossbuild.go

```diff
@@ -99,19 +107,25 @@ func CrossBuild(options ...CrossBuildOption) error {
 
 	log.Println("crossBuild: Platform list =", params.Platforms)
 	var deps []interface{}
+	xpack := []bool{false}
+	if params.WithXPack {
+		xpack = append(xpack, true)
+	}
 	for _, buildPlatform := range params.Platforms {
 		if !buildPlatform.Flags.CanCrossBuild() {
 			return fmt.Errorf("unsupported cross build platform %v", buildPlatform.Name)
 		}
 
-		builder := GolangCrossBuilder{buildPlatform.Name, params.Target}
-		if params.Serial {
-			if err := builder.Build(); err != nil {
-				return errors.Wrapf(err, "failed cross-building target=%v for platform=%v",
-					params.Target, buildPlatform.Name)
+		for _, xpack := range xpack {
```

Comment:
I first stumbled over this line of code and was thinking "why is there a list of x-pack options"? I now get that  it either has 1 or 2 entries for with and without x-pack. I wonder if a better name for the slice would be buildOptions or similar??